parse-jtl.py

```diff
@@ -54,6 +56,10 @@ def parse_options():
                         action="store_true",
                         dest="pretty_print",
                         help="Parse and pretty print")
+    parser.add_argument("--generate-histograms",
+                        action="store_true",
+                        dest="generate_histograms",
+                        help="Generate a PDF file with charts of the results.")
```

Comment:
It is necessary to use `--generate-histograms` with some other argument (-c, -p, etc.) now.

I think we can omit this need adding more checks at line 89.